addons/crm/static/src/views/fill_temporal_service.js

```diff
@@ -102,15 +90,15 @@ export class FillTemporalPeriod {
         this._computeEnd();
     }
     /**
-     * Compute the moment for the start of the period containing "now"
+     * Compute the luxon Datetime for the start of the period containing "now"
      *
      * @private
      */
     _computeStart() {
-        this.start = GRANULARITY_TABLE[this.granularity].startOf(moment());
+        this.start = GRANULARITY_TABLE[this.granularity].startOf(luxon.DateTime.now());
     }
     /**
-     * Compute the moment for the end of the fill_temporal period. This bound is exclusive.
+     * Compute the luxon Datetime for the end of the fill_temporal period. This bound is exclusive.
```

Comment:
I admit it's not that clear since it's a method that returns nothing, but the purpose of `_computeEnd` is to set the value for `this.end` which is indeed a `Datetime` (But I used the /** */ syntax for a comment inside a function, maybe I can change that back to // instead)